snapcraft/services/assertions.py

```diff
@@ -161,7 +181,10 @@ def _edit_yaml_file(self, filepath: pathlib.Path) -> models.EditableAssertion:
                     data=data,
                     # filepath is only shown for pydantic errors and snapcraft should
                     # not expose the temp file name
-                    filepath=pathlib.Path(self._assertion_name.replace(" ", "-")),
+                    filepath=pathlib.Path(self._assertion_name),
+                )
+                craft_cli.emit.progress(
+                    f"Edited {self._assertion_name}.", permanent=True
```

Comment:
I want something to stick in `brief` mode.  `edit-validation-sets` isn't a great reference because nothing is permanent.

The current UX looks like:
```
>  snapcraft staging servers> snapcraft edit-registries ...
Retrieved registries set 'test-mr-cal-1' from the store.
Edited registries set.
Built registries set
Signed assertion.
Successfully edited registries set 'test-mr-cal-1'
```

And with a syntax error and a business logic error:
```
>  snapcraft staging servers> snapcraft edit-registries dFFHwNHZmpzDv09t5jHH8WfteBN8ciI4 test-mr-cal-1 --key-name test6
Retrieved registries set 'test-mr-cal-1' from the store.
Bad registries set content:
- input should be 'read', 'write' or 'read-write' (in field 'views.wifi-setup.rules[0].access')
Do you wish to amend the registries set? [y/N]: y
Edited registries set.
Built registries set
error: cannot assemble assertion registry: cannot define view "wifi-setup": storage path "foo.ssids" for request "ssids" is invalid after "": cannot use "foo" as key in map
failed to sign assertion
Do you wish to amend the registries set? [y/N]: y
Edited registries set.
Built registries set
Signed assertion.
Successfully edited registries set 'test-mr-cal-1'.
```

Do you think everything except the last message (`Successfully edited registries set 'test-mr-cal-1'`) should be ephemeral?

Reply:
I wanted something to stick, but I may have made too many messages permanent.  `edit-validation-sets` has no output, so it's not a good reference. Perhaps everything but the last message should be temporary?


current ux:
```bash
>  snapcraft staging servers> snapcraft edit-registries ...
Retrieved registries set 'test-mr-cal-1' from the store.
Edited registries set.
Built registries set
Signed assertion.
Successfully edited registries set 'test-mr-cal-1'.
```

with a syntax error and a signing error:
```bash
>  snapcraft staging servers> snapcraft edit-registries ...
Retrieved registries set 'test-mr-cal-1' from the store.
Bad registries set content:
- input should be 'read', 'write' or 'read-write' (in field 'views.wifi-setup.rules[0].access')
Do you wish to amend the registries set? [y/N]: y
Edited registries set.
Built registries set
error: cannot assemble assertion registry: cannot define view "wifi-setup": storage path "foo" for request "ssids" is invalid after "": cannot use "foo" as key in map
Failed to sign assertion
Do you wish to amend the registries set? [y/N]: y
Edited registries set.
Built registries set
Signed assertion.
Successfully edited registries set 'test-mr-cal-1'.
```

Reply:
yes, only the last one should be permanent

Reply:
I would even change it a bit, to "Successfully create revision {revision!r} on sequence {sequence!r} for set {validation_set!r}"